images/nbserve/nginx.py

```diff
@@ -1,8 +1,11 @@
-#!/usr/bin/python
+#!/usr/bin/python3
 import os
 
 
 CONFIG = """
+# Load modules
+include /etc/nginx/modules-enabled/*.conf;
```

Comment:
I'm not sure why things worked on Jessie without this directive.

Reply:
That explains why it didn't work when I rebuilt it on buster! It is currently running: https://github.com/crookedstorm/paws/tree/master/images/nbserve
I built that image and put it in the Toolforge-based paws-public setup.  It's similar except that instead of using the debian package, I ran my own compile based on the openresty official image and added in the fancyindex module.

I created that version for use in the new cluster. Basically all the new stuff for the new cluster is over in my fork until I actually test it in the new cluster and put in a PR.

Reply:
I actually have suspected that the version running on the cluster was never synced perfectly to the version in GitHub, and there was a local change somewhere that probably included this directive.  I wasn't sure what directive was missing so I just went ahead and recompiled. :)